addons/im_livechat/static/src/core/web/channel_member_list_patch.xml

```diff
@@ -7,9 +7,9 @@
                         <i class="fa fa-comment-o me-1" aria-label="Lang"/>
                         <t t-esc="member.getLangName()"/>
                     </span>
-                    <span t-if="member.persona?.country or props.thread.country_id">
+                    <span t-if="member.persona?.country_id or props.thread.country_id">
                         <i class="fa fa-globe me-1" aria-label="country"/>
-                        <t t-esc="member.persona?.country?.name ?? props.thread.country_id.name"/>
+                        <t t-esc="member.persona?.country_id?.name ?? props.thread.country_id.name"/>
```

Comment:
Country on personal model wasn't updated.

I think we also have missing test coverage here, because this shouldn't be working (field not defined properly = can't follow relations to rad the name).

Reply:
country also not updated in `correspondentCountry` which means usage of `correspondentCountry` are also lacking test coverage